pkg/reconciler/pipelinerun/pipelinerun_test.go

```diff
@@ -8787,3 +8882,107 @@ spec:
 status:
   startTime: %s`, prName, specStatus, now.Format(time.RFC3339)))
 }
+
+func shouldHaveFullEmbeddedStatus(embeddedVal string) bool {
+	return true
+	// TODO(abayer): Uncomment when implementation of TEP-0100 is done.
+	// return embeddedVal == config.FullEmbeddedStatus || embeddedVal == config.BothEmbeddedStatus
+}
+
+func shouldHaveMinimalEmbeddedStatus(embeddedVal string) bool {
+	return false
+	// TODO(abayer): Uncomment when implementation of TEP-0100 is done.
+	// return embeddedVal == config.MinimalEmbeddedStatus || embeddedVal == config.BothEmbeddedStatus
+}
+
+func verifyTaskRunStatusesCount(t *testing.T, embeddedStatus string, prStatus v1beta1.PipelineRunStatus, taskCount int) {
+	if shouldHaveFullEmbeddedStatus(embeddedStatus) && len(prStatus.TaskRuns) != taskCount {
```

Comment:
rather than creating these functions "shouldhavefullembeddedstatus" etc I think it might be cleaner to not have these branches at all, and just add them when the embedded status functionality is enabled.

Reply:
I’m not sure what you mean? The point of this PR is to cut down on the size of the implementation PR, so if we don’t add the new test behavior for the embedded status functionality here, we have to do it in the implementation PR instead.

Reply:
ok-- whatever you think is best here, although my intention isn't specifically to reduce number of lines of code but number of "things" that a given PR is doing

Reply:
Fair enough. =) I guess I just feel like stubbing `shouldHaveFullEmbeddedStatus` and `shouldHaveMinimalEmbeddedStatus` here with the `TODO`s, and full implementations of `verify...` is clearer on the intent, particularly since there (hopefully!) will not be a long gap between this PR landing and the implementation PR landing.